elm/wizard.py

```diff
@@ -652,10 +655,13 @@ def _format_refs(self, refs, ids):
                                    "connection or query.")
 
         unique_ref_list = []
+        unique_nrel_ids = set()
         for ref_dict in ref_list:
-            if any(ref_dict == d for d in unique_ref_list):
+            if ref_dict['nrel_id'] in unique_nrel_ids:
```

Comment:
what do you think about this? We need the chunk id column `id` to order the reference list and the document id column `nrel_id` to make sure each item in the list is unique. All of our DBs currently use `nrel_id` but I could change it to `document_id` to make it more public friendly, it would just require re-creating meta files and a little additional work. Or if you have another method in mind I am all ears.